inc/class-wpseo-utils.php

```diff
@@ -1102,44 +1102,27 @@ public static function get_analysis_worker_log_level() {
 	}
 
 	/**
-	 * Returns the home url with the following modifications:
+	 * In case WPML is installed, returns the original home_url and not the WPML version.
```

Comment:
To the documentation I would add a sentence above saying something like `Returns the home url`. Now it seems as if this function only is relevant if WPML is installed.